### ISSUE TITLE
Project 1 Feedback

### ISSUE THREAD

**Issue by mattstuddert** [madgoose/ga-javascript-development#3]
Amazing work on this Dom. The game looks awesome!

I was happy to see that you went for the Handlebars loop in order to generate the markup for the cards :) Great work doing this game in vanilla JS. You'll definitely get a deeper understanding of the language by doing so! 

As far as your code goes, there's not much to actually say other than keep going as you are. It is obvious that you have got a very solid grasp of some fundamental concepts of JS. The fact that you're using a class and then adding methods to that class in order to execute the program shows this. 

I know it is not the finished article, so do feel free to give me a heads up when it's finished and I'll take another look. 

Excellent work on this and keep it up!

Let me know if you have any questions at all about any issues you had.

**Comment by madgoose** [comment]
thanks matt :-) 
tl;dr carl jung had his concept of synchronicity. as i am "between contracts" i've had free time on my hands to fully immerse myself in this project!
